frontend/kesaseteli/employer/public/locales/sv/common.json

```diff
@@ -11,6 +11,14 @@
     "retry": "SV Lataa sivu uudelleen",
     "logout": "SV Kirjaudu ulos"
   },
+  "loginPage": {
+    "logoutLabel": "Olet kirjautunut ulos",
```

Comment:
logoutLabel and loginLabel are duplicates, you could use the same keys for them as the header does